src/main/java/org/opentripplanner/model/calendar/openinghours/OpeningHours.java

```diff
@@ -72,4 +88,18 @@ public boolean isOpen(int day, int secondsSinceMidnight) {
       days.get(day) && this.startTime <= secondsSinceMidnight && secondsSinceMidnight <= endTime
     );
   }
+
+  private static String toOsm(String description) {
+    return switch (description.toLowerCase()) {
+      case "business days" -> "Mo-Fr";
+      case "monday" -> "Mo";
+      case "tuesday" -> "Tu";
+      case "wednesday" -> "We";
+      case "thursday" -> "Th";
+      case "friday" -> "Fr";
+      case "saturday" -> "Sa";
+      case "sunday" -> "Su";
+      default -> description;
+    };
+  }
```

Comment:
The `String osmFormat()` creates a weak dependency from the OpeningHours class on OSM - and doing so is not part of the responsibility of the any model class. Did you consider creating a `OsmOpeningHoursSupport` class outside the model, perhaps inside the `osm` package for this?

Reply:
Another thing related to this method, I think this logic only successfully changes the description in simple cases where the description is just one word. The description can be pretty much anything (although the current implementations only produces certain variations). I think we should document in javadoc the limitations of this method.

Reply:
> The `String osmFormat()` creates a weak dependency from the OpeningHours class on OSM - and doing so is not part of the responsibility of the any model class. Did you consider creating a `OsmOpeningHoursSupport` class outside the model, perhaps inside the `osm` package for this?

Yes, I can do that. Could you tell exactly which package you mean. There are a couple of `osm` packages and none is, IMO, a great fit.

> Another thing related to this method, I think this logic only successfully changes the description in simple cases where the description is just one word. The description can be pretty much anything (although the current implementations only produces certain variations). I think we should document in javadoc the limitations of this method.

That is correct. I will document the shortcomings.

If we want to have a robust conversion then we will need to keep a lot more data in the model.

Reply:
@t2gran @optionsome I've taken your feedback into account now and will move the class once I get the final package name.

Reply:
> Could you tell exactly which package you mean.

I am sorry, I do not have a good anser. The problem is that we do not have a good module structure. Ideally we should have *domain modules* and the some *services/use-case* support modules on top. I think we all need to work with this and try to reduce the dependencies and making encapsulated modules with low coupling.  

It is ok for now, since it in this case only have an indirect dependency and now is separated into its own class.